ethcore/types/src/verification.rs

```diff
@@ -57,7 +57,7 @@ impl VerificationQueueInfo {
 }
 
 /// An unverified block.
-#[derive(PartialEq, Debug, MallocSizeOf)]
+#[derive(Clone, PartialEq, Debug, MallocSizeOf)]
```

Comment:
`Clone` is used for benchmarks/tests? 

Why not `#[cfg_attr(any(foo, bar), derive(Clone)]`?

Reply:
Yes, that is the reason. It's more convenience than necessity though. In general though, why is it bad to derive `Clone`? Reading https://rust-lang-nursery.github.io/api-guidelines/interoperability.html#c-common-traits it seems idiomatic to do so whenever it's convenient?

Reply:
nothing bad I guess but it might have the reasoning that such big structures should not be  cloned which are enforced by `rustc` but I don't know really just guessing!